src/Microsoft.AspNetCore.TestHost/WebSocketClient.cs

```diff
@@ -87,6 +87,7 @@ private class RequestState : IDisposable, IHttpWebSocketFeature
         {
             private readonly IHttpApplication<Context> _application;
             private TaskCompletionSource<WebSocket> _clientWebSocketTcs;
+            private CancellationTokenRegistration _cancellationTokenRegistration;
```

Comment:
Not sure what _cancellationTokenRegistration is used for. I don't see anywhere in this PR where the callback is deregistered. I think it's safe to remove?